apps/extension/src/service-worker.ts

```diff
@@ -52,18 +54,25 @@ const servicesConfig = {
 const services = new Services(servicesConfig);
 await services.initialize();
 
-// this only handles stdClient requests
-chrome.runtime.onMessage.addListener(
-  (
-    message: unknown,
-    _: chrome.runtime.MessageSender,
-    sendResponse: (response: unknown) => void,
-  ) => {
-    if (!isStdRequest(message)) return;
-    stdRouter(message, sendResponse, services);
-    return true;
-  },
-);
+chrome.runtime.onMessage.addListener((message, sender, sendResponse) => {
+  if (sender.id !== chrome.runtime.id) return; // unhandled
+  switch (message) {
+    case 'PENUMBRA_CLEAR_CACHE':
+      void services.clearCache().then(
+        () => sendResponse(true),
+        () => sendResponse(false),
+      );
+      return true; // async handler
+    case 'PENUMBRA_SYNC_BLOCKS':
+      void services.tryToSync().then(
+        () => sendResponse(true),
+        () => sendResponse(false),
+      );
+      return true; // async handler
```

Comment:
Is there a way we can do this without every method doing .then() + sendResponse? Maybe a wrapper for this? Think the old std router did that.